src/templates/app/user/metatrader.jsx

```diff
@@ -216,6 +216,17 @@ const Metatrader = () => (
                                 it.L('Negative balance protection'),
                             ]}
                         />
+                        <AccountDesc
+                            account_type={['labuan_advanced']}
+                            title={it.L('Advanced Account')}
+                            description={it.L('Our MetaTrader 5 Advanced account provides you with tight spreads, higher ticket size and offers more products.')}
+                            items={[
+                                it.L('Leverage up to [_1]', '1:100'),
+                                it.L('Variable spreads'),
+                                it.L('Market execution'),
+                                it.L('No commission'),
+                            ]}
+                        />
                         <AccountDesc
                             account_type={['vanuatu_advanced']}
```

Comment:
Do we still have `vanuatu_advanced` or it's changed to `labuan_advanced`?

Reply:
It's going to be changed to `labuan_advanced` but I left it there for the time being until we get better confirmation to how we handle the transition from `vanuatu` to `labuan` for advanced account creation. I'll comment out `vanuatu_advanced` for now. Thanks